src/hooks/useEthBridges.js

```diff
@@ -0,0 +1,39 @@
+import { useCall, useEthers, useToken, useTokenBalance } from "@usedapp/core";
+import { Contract } from "ethers";
+import BridgeABI from "../assets/abis/Bridge.json";
+
+export const ASSETS = ["LLM", "LLD"];
+const ADDRESSES = {
+    LLM: process.env.REACT_APP_LLM_BRIDGE,
+    LLD: process.env.REACT_APP_LLD_BRIDGE,
+};
+
+function useEthBridge(asset) {
+    // FIXME pass this through redux?
```

Comment:
What about this ?

Reply:
To keep codebase consistent it would make sense here to pass this through whole redux flow:
* create action to `getEthBridge`
* create sagas to catch this and create everything (basically this function refactored for sagas)
* create selector for getting this in views

This is a code-quality thing, not functional.